install.js

```diff
@@ -24,6 +29,12 @@ var app = builder();
 var masterTenant = false;
 var superUser = false;
 
+var isVagrant = function () {
+  if (process.argv.length != 2) {
+      return true;
+    }
```

Comment:
minor indentation issue here. also, an explicit `return false;` would be a bit clearer:

````
if (process.argv.length != 2) {
      return true;
}

return false;
````